Automattic/pocket-casts-ios
# [Slumber] Add to the list of features

Adds the Slumber Studios feature to the list of available features in:

1. Upsell screen
2. Prompt screen under account

https://github.com/Automattic/pocket-casts-ios/assets/7040243/008b280d-bf59-47b8-aeb8-219a3e8ae380

## To test

1. Run the app
2. Go to Pofile > Beta Features > make sure `slumber` is toggled ON
3. Login to an account without Plus
4. Go to the "Podcasts" tab
5. Tap the folder locked icon
6. ✅ You should see a mention to Slumber under Plus
7. Tap "Slumber Studios"
8. ✅ Their website should open inside the app. Meaning you won't leave to Safari
9. Dismiss the website and the plus modal
10. Go to Profile > Account
11. ✅ You should a mention to Slumber under Plus
12. Tap "Slumber Studios"
8. ✅ Their website should open inside the app. Meaning you won't leave to Safari
9. Swipe to Patron
10. ✅ You should see the gratitude message (this was added to keep both Plus and Patron of about the same size)
11. Go to Pofile > Beta Features > make sure `slumber` is toggled OFF
12. Re-do the tests, you should not see a mention to Slumber

## Checklist
